### ISSUE TITLE
source files have the same name -- but one of them does not exist

### ISSUE THREAD

**Issue by nilsbecker** [the-lambda-church/merlin#453]
i am getting this error when using locate on CCHashtbl from `containers`:

     Several source files in your path have the same name: 

and then the path is _exactly_ the same, in my opam install location, but the two file names are `CCHashtbl.mli` and `cCHashtbl.mli`. the second one does not exist at that location.

**Comment by def-lkb** [comment]
I assume you are using OSX? Then it is likely both exists :).

OSX has a weird fs semantics, sometimes described as "case insensitive but case aware", which confuses Merlin. This will be fixed...

**Comment by nilsbecker** [comment]
yes, it's os x. fwiw, the directory `~/.opam/4.02.3/lib/containers` contains really only one file matching `??Hashtbl.mli`.

**Comment by dbuenzli** [comment]
Related to #425

**Comment by nilsbecker** [comment]
wait, now after installing from `#prerelease` i cannot reproduce it anymore. it may be already fixed.

**Comment by nilsbecker** [comment]
ok, now invoking `MerlinLocate` on `CCHashtbl` does indeed work. what still does not work is invoking `MerlinLocate` on `CCHashtbl.Make` (ie on `Make`). the error message is that the ML file for CCHashtbl cannot be found. but it would be ok to just look up the MLI file just as for the module? this may be unrelated
